packages/shared/lib/src/sdk/mod.rs

```diff
@@ -131,9 +238,20 @@ impl Sdk {
     ) -> Result<(), JsError> {
         let args = tx::ibc_transfer_tx_args(tx_msg, password)?;
 
-        namada::ledger::tx::submit_ibc_transfer(&self.client, &mut self.wallet, args)
+        let (mut tx, _, pk) =
+            namada::ledger::tx::build_ibc_transfer(&self.client, &mut self.wallet, args.clone())
+                .await
+                .map_err(|e| JsError::from(e))?;
+
+        signing::sign_tx(&mut self.wallet, &mut tx, &args.tx, &pk)
+            .await
+            .map_err(|e| JsError::from(e))?;
+
+        namada::ledger::tx::process_tx(&self.client, &mut self.wallet, &args.tx, tx)
```

Comment:
Usually I'm not the biggest fan of this, but maybe it's worth adding a function like `sign_and_process_tx` :stuck_out_tongue_closed_eyes:  as it seems that sign and process are often called together

Reply:
This is now done!